.rubocop.yml

```diff
@@ -0,0 +1,52 @@
+AllCops:
+    Exclude:
+    - "Guardfile"
+    - "Rakefile"
+    
+    DisplayCopNames: true
+    
+    Layout/LineLength:
+    Max: 120
+    Metrics/MethodLength:
+    Max: 40
+    Metrics/AbcSize:
+    Max: 50
+    Metrics/ClassLength:
+    Max: 150
+    Metrics/BlockLength:
+    ExcludedMethods: ['describe']
+    Max: 30
+    
+    
+    Style/Documentation:
+    Enabled: false
+    Style/ClassAndModuleChildren:
+    Enabled: false
+    Style/EachForSimpleLoop:
+    Enabled: false
+    Style/AndOr:
+    Enabled: false
+    Style/DefWithParentheses:
+    Enabled: false
+    Style/FrozenStringLiteralComment:
+    EnforcedStyle: never
+    Style/FrozenStringLiteralComment:
+        Enabled: false
```

Comment:
- [ ] These lines are different in the original, remember you should not modify the original rubocop file, please use the original, you can find it here: https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml
![image](https://user-images.githubusercontent.com/53256139/94779950-50744000-038d-11eb-9dfc-06992f96a0a5.png)